### ISSUE TITLE
3 way merge editor checkboxes not visible

### ISSUE THREAD

**Issue by matusaurio** [Monokai/monokai-pro-vscode#307]
Recently, it was introduced the [3 way merge editor](https://code.visualstudio.com/updates/v1_69#_3-way-merge-editor), but the checkboxes to accept and combine changes are not visible.

**Comment by phyllisstein** [comment]
As a temporary fix, you can add this to your VSCode settings:

```json
{
  "workbench.colorCustomizations": {
    "[Monokai Pro]": {
      "dropdown.border": "#221f22"
    }
  }
}
```
<img width="101" alt="CleanShot 2022-08-01 at 17 32 03@2x" src="https://user-images.githubusercontent.com/2207980/182250137-f78fd3d8-592c-4aeb-b9ff-8d1d074ea1e8.png">

**Comment by fgb4101** [comment]
> As a temporary fix, you can add this to your VSCode settings:
> 
> ```json
> {
>   "workbench.colorCustomizations": {
>     "[Monokai Pro]": {
>       "dropdown.border": "#221f22"
>     }
>   }
> }
> ```

I see also:
```json
"checkbox.border": "#525252"
```

**Comment by phyllisstein** [comment]
Well spotted! I _slightly_ prefer the darker color, since it provides some contrast between the merge flag and the box itself. Hopefully we’ll get an official ruling soon.

<img width="1212" alt="CleanShot 2022-08-15 at 07 55 47@2x" src="https://user-images.githubusercontent.com/2207980/184631036-f48ccf9d-85e2-447a-a4a5-cca360dc3bd3.png">

**Comment by jacksongoode** [comment]
Bumping this!

**Comment by Monokai** [comment]
fixed in 1.1.21
